src/app/components/layouts/expandable-text/expandable-text.component.html

```diff
@@ -0,0 +1,14 @@
+<div id="{{id}}" *ngIf="!expanded; else expandedTemplate" [clamp]="lines">
+    <ng-content *ngTemplateOutlet="expandedTemplate">
```

Comment:
ich glaube `ng-container` wäre hier besser
```suggestion
    <ng-container *ngTemplateOutlet="expandedTemplate">
```